vignettes/dependencies.Rmd

```diff
@@ -0,0 +1,86 @@
+---
+title: "Devtools Remote Dependencies"
```

Comment:
Could we just make this "Devtools dependencies" and include remote versions as a section?